src/rgw/rgw_rados.cc

```diff
@@ -3201,6 +3200,7 @@ void RGWRados::finalize()
   if (cr_registry) {
     cr_registry->put();
   }
+  delete meta_mgr;
```

Comment:
although jewel don't enter this process when reopen_logs, but it may hit this problem when the RGWRealmReloader::reload() fuction is called(for example,  change the master zone to a different zone).